src/prem/assignment29/SecondMaxiumNumber.java

```diff
@@ -0,0 +1,37 @@
+package prem.assignment29;
+
+import java.util.Arrays;
+
+public class SecondMaxiumNumber {
+	
+    /*Assignment - 29 : 26th Nov'2020 
+	Find second maximum from array without sorting the array.
+	int arr[] = { 25,36,87,60,9,8, 10};
+	*/
+
+
+		void secondMaxNum(int arr[]) {
+
+			int max = arr[0];
+			int secondMax = arr[1];
+			for (int index = 0; index < arr.length; index++) {
+				for (int innerIndex = 2; innerIndex < arr.length; innerIndex++) {
+					if (arr[innerIndex] > max) {
+						secondMax = max;
+						max = arr[innerIndex];
+					} else if (arr[innerIndex] > secondMax)
```

Comment:
you should one extra condition for max number is like

else if (input[index] > secondMax && input[index] != max)